app/scripts/services/dom-event.js

```diff
@@ -59,7 +59,11 @@ const register = pubSub => (event, newElement, useCapture = false) => {
   registeredEls[event] = newElement;
   registeredEls[event].__handler__ = getEventHandler(event, pubSub);
   registeredEls[event].addEventListener(
-    event, registeredEls[event].__handler__, useCapture
+    event, registeredEls[event].__handler__,
+    {
+      capture: useCapture,
+      passive: false,
```

Comment:
Just out of curiosity, is this turning all captured events into active events? I haven't used that feature before that's why I am asking :)

Reply:
Yes, that was the default until Google changed it for `wheel` and `mousewheel` events in Chrome 73.